tiledb/sm/filter/test/add_1_in_place_filter.cc

```diff
@@ -44,6 +44,10 @@ void Add1InPlace::dump(FILE* out) const {
   (void)out;
 }
 
+void Add1InPlace::output(std::string* out) const {
+  (void)out;
+}
```

Comment:
FYI

Omitting the argument name is legal syntax. It means you don't need a useless void-cast statement; the body can be empty. Such functions should be `inline` in the header. Having a separate body definition is junk syntax.